### DISCUSSION TITLE
Welcome to Language-Tutorials Discussions!

### DISCUSSION THREAD

**Discussion by whippingdot** [whippingdot/Language-Tutorials#5] [General]
# 👋 Welcome!
Discussions is enabled and I hope it is useful. tell me what improvements I need in my courses, if you want to join and help me make the courses, or help me fix mistakes in my courses. You can either use dicussions, a PR, or issues, anything is fine by me, but I would recommend discussions as it is the easiest for me. This will be used for:
  * Telling me what improvements I need in my courses
  * Join requests
  * Helping me fix mistakes in my courses

**Comment by kokonut27** [comment]
Uh, can I add to the python page? So there's a `sep`, so I wanted to add an `end`. Can I? Plz reply lol

**Reply by darkdarcool** [reply]
you probably can @JBYT27!(i would call you jb, but I needed to ping you.)

**Reply by kokonut27** [reply]
Lol, ok! Thnx

**Reply by whippingdot** [reply]
u guys can. Ima make a replit post on this today inviting people to help work on this. It isn't finished yet so it isn't a share post. I will tell you the details. The main thing is try to follow the way my files look when you add things. dardarcool's ruby is completely different from mine, which is ok by me, but I hope everyone else's is similar to mine.

**Reply by darkdarcool** [reply]
yeah, make sure to ask people for how their learning style is so I can (try) to make lessons targeted to them.

**Reply by kokonut27** [reply]
ok

**Comment by darkdarcool** [comment]
@whippingdot, ruby md's are going good, but I want someone to cover ruby on rails, because it is a big and important tool(~~which is why I don't know it~~), so I would like it if you could find someone to cover it? If not, we can skip it.

Cheers!(~~I love ending my comments like that lul, I feel British even though im American~~)

**Reply by whippingdot** [reply]
lul. Also I don't know anyone who can cover it, but if this blows up we surely will find someone. @BD103 @JBYT27 @Rayyan06 do any of u know ruby on rails?

**Reply by darkdarcool** [reply]
kk good.

**Reply by BD103** [reply]
If ima be honest, I don't know any Ruby.

**Reply by darkdarcool** [reply]
lol maybe you should read my documentation on it here

**Reply by BD103** [reply]
HAH! Good point

**Reply by Rayyan06** [reply]
*sigh* I have no idea about ruby on rails. I know Django though (web framework for python) quite well so I can come in useful if we are writing stuff about that. I hope this project blows up and we get a ton of contributors.

**Reply by kokonut27** [reply]
lol

**Comment by whippingdot** [comment]
@darkdarcool @JBYT27 @BD103 @Rayyan06 and @CrimsonCodes0 bruh comeback time baby less goooo

Lets continue from the github universe 2021 date, 27th of October. I don't know if I will actually be free then, but let us revive this project already!!!

**Reply by BD103** [reply]
Github universe?

**Reply by darkdarcool** [reply]
ok?

**Reply by whippingdot** [reply]
> ok?

Why the question mark dark?

**Reply by whippingdot** [reply]
> Github universe?

Yea, it is in like a week I guess

**Reply by darkdarcool** [reply]
> > ok?
> Why the question mark dark?

Just a little unsure, I'm making a build system and an actual PL right now(with parser, lexer, etc), not sure when I'll be able to carve out time, but I'll try

**Reply by kokonut27** [reply]
Same, coding a search program rn

**Reply by BD103** [reply]
Yeah, I'd love to help but I'm working on one of the biggest projects I've ever taken on yet.
